Src/SwqlStudio/ObjectExplorer/DocumentationBuilder.cs

```diff
@@ -0,0 +1,141 @@
+﻿using System;
+using System.Text;
+using System.Windows.Forms;
+using SwqlStudio.Metadata;
+
+namespace SwqlStudio
+{
+    internal class DocumentationBuilder
+    {
+        public static Documentation Build(TreeNode node)
+        {
+            var data = node.Tag;
+            var connectedNode = node as TreeNodeWithConnectionInfo;
+            var provider = connectedNode?.Provider;
+
+            if (provider == null)
+                return Documentation.Empty;
+
+            if (data is SwisMetaDataProvider)
+                return ProviderDocumentation(provider);
+
+            if (data is string nameSpace)
```

Comment:
Should we have a class for namespaces?  Is `string` sufficient?

Reply:
There is no logic behind the namespace now, is string is still enough